.travis/mypy

```diff
@@ -37,22 +37,23 @@ tmp="$(mktemp -t mypy.XXXX)";
-        -e '^src/klein/_plating.py:.* error: Function is missing a type annotation'                      \
-        -e '^src/klein/_plating.py:[0-9:]*: error: Need type annotation for '                            \
-        -e '^src/klein/_resource.py:[0-9:]*: error: Function is missing a type annotation'               \
-        -e '^src/klein/test/_trial.py:[0-9:]*: error: Function is missing a type annotation'             \
-        -e '^src/klein/test/py3_test_resource.py:[0-9:]*: error: Function is missing a type annotation'  \
-        -e '^src/klein/test/test_app.py:[0-9:]*: error: Function is missing a type annotation'           \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Function is missing a type annotation'       \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Need type annotation for '                   \
-        -e '^src/klein/test/test_resource.py:[0-9:]*: error: Function is missing a type annotation'      \
-        -e '^src/klein/test/util.py:[0-9:]*: error: Function is missing a type annotation'               \
+        -e ': error: Argument "converter" to "attrib" has incompatible type'                                \
+        -e ': error: Method must have at least one argument'                                                \
+        -e ': error: The type alias to [A-Za-z]* is invalid in runtime context'                             \
+        -e ': note: Use "-> None" if function does not return a value'                                      \
+        -e '^src/klein/_app.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'                    \
+        -e '^src/klein/_decorators.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'             \
+        -e '^src/klein/_iapp.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'                   \
+        -e '^src/klein/_plating.py:.* error: Function is missing a [a-z ]* annotation'                      \
+        -e '^src/klein/_plating.py:[0-9:]*: error: Need type annotation for '                               \
+        -e '^src/klein/_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'               \
+        -e '^src/klein/test/_trial.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'             \
+        -e '^src/klein/test/py3_test_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'  \
+        -e '^src/klein/test/test_app.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'           \
+        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'       \
+        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Need type annotation for '                      \
+        -e '^src/klein/test/test_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'      \
+        -e '^src/klein/test/util.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'               \
```

Comment:
This error message comes in two flavors now ("…missing a type…") and ("…missing a return type…"); we still want to ignore both for now.